public/components/model/model_versions_panel/model_versions_panel.tsx

```diff
@@ -64,11 +74,21 @@ const getStatesParam = ({
   });
 };
 
+const getSortParam = (sort: Array<{ id: string; direction: 'asc' | 'desc' }>) => {
+  const id2fieldMap: { [key: string]: string } = {
+    lastUpdatedTime: 'last_updated_time',
```

Comment:
Q: do you need to have mapping for `lastRegisteredTime` `lastDeployedTime`, etc?

Reply:
No need for now, we don't have such columns.